style.css

```diff
@@ -77,3 +79,178 @@ ul {
 li {
   padding-right: 20px;
 }
+
+.card {
+  background-color: #fff;
+  border-radius: 16px;
+  border: 1px solid #dfe1e6;
+  padding: 20px 0;
+}
```

Comment:
- [ ] Please make sure to follow the Figma template as close as possible, your margins are different from the ones in the Figma template

## Figma
![image](https://user-images.githubusercontent.com/28109626/148598139-6b4976f8-40c1-4aae-a49c-144b615a410b.png)

## Your Project
![image](https://user-images.githubusercontent.com/28109626/148598069-2ca97906-5b0e-40e1-86a1-82370e798c8e.png)